gamemodes/terrortown/gamemode/client/vgui/cl_sb_row.lua

```diff
@@ -13,16 +13,16 @@ local IsValid = IsValid
 local surface = surface
 local vgui = vgui
 
-local color_trans = Color(0, 0, 0, 0)
+local colorTransparent = Color(0, 0, 0, 0)
 
-local ttt2_indicator_dev = "vgui/ttt/ttt2_indicator_dev"
-local ttt2_indicator_vip = "vgui/ttt/ttt2_indicator_vip"
-local ttt2_indicator_addondev = "vgui/ttt/ttt2_indicator_addondev"
-local ttt2_indicator_admin = "vgui/ttt/ttt2_indicator_admin"
-local ttt2_indicator_streamer = "vgui/ttt/ttt2_indicator_streamer"
-local ttt2_indicator_heroes = "vgui/ttt/ttt2_indicator_heroes"
+local materialIndicatorDev = "vgui/ttt/materialIndicatorDev"
+local materialIndicatorVIP = "vgui/ttt/materialIndicatorVIP"
+local materialIndicatorAddonDev = "vgui/ttt/materialIndicatorAddonDev"
+local materialIndicatorAdmin = "vgui/ttt/materialIndicatorAdmin"
+local materialIndicatorStreamer = "vgui/ttt/materialIndicatorStreamer"
+local materialIndicatorHeroes = "vgui/ttt/materialIndicatorHeroes"
```

Comment:
How about putting it into a list with function checks and a layering order to cut code complexity down? (another PR)